dist/build.gradle

```diff
@@ -128,3 +137,12 @@ task('distZip', type: Zip) {
 distribution {
     dependsOn(distZip)
 }
+
+// Add a task in the root project that collects all the dependencyReport data for each project
```

Comment:
should this task be in the root project, or should the comment be updated ?

Reply:
Comment update for sure.